var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -343,6 +349,7 @@ class Lammps(CMakePackage, CudaPackage, ROCmPackage):
     )
```

Comment:
Did you want to remove all these deprecated versions while you're at it?

Reply:
LAMMPS is one of those codes where having the capability of installing an older version for reproducibility can be of interest. I have yet to find a good metric to determine when I should discard a specific LAMMPS version.

Reply:
Usually the only reason to deprecate something is so it can be removed. You could also un-deprecate them.

Reply:
Un-deprecation would defeat the purpose since we don't actually want people using those versions anymore unless they know what they are doing.